### ISSUE TITLE
Find BigDecimal implementation

### ISSUE THREAD

**Issue by sublimator** [ripple/ripple-dot-net#9]

**Comment by sublimator** [comment]
https://sine.codeplex.com/

**Comment by matejr** [comment]
BigRational in Math.NET: 

http://numerics.mathdotnet.com/

Why exactly do you need BigDecimal? For AmountValue?

**Comment by sublimator** [comment]
Yeah, with the rounding modes would be nice

**Comment by matejr** [comment]
When do you need rounding? Just when converting to/from string or after each operation?

**Comment by sublimator** [comment]
Yeah, for arithemtic etc. The AmountValue is a hack for serialisation until
something better comes along
